torchtext/csrc/register_torchbindings.cpp

```diff
@@ -122,10 +123,27 @@ TORCH_LIBRARY_FRAGMENT(torchtext, m) {
             return _deserialize_vocab(states);
           });
 
+  m.class_<GPT2BPEEncoder>("GPT2BPEEncoder")
+      .def(torch::init<c10::Dict<std::string, int64_t>,
+                       c10::Dict<std::string, int64_t>, std::string,
+                       c10::Dict<int64_t, std::string>, bool>())
+      .def("encode", &GPT2BPEEncoder::Encode)
+      .def_pickle(
+          // __getstate__
+          [](const c10::intrusive_ptr<GPT2BPEEncoder> &self)
+              -> GPT2BPEEncoderStatesTorchbind {
+            return _serialize_gpt2_bpe_encoder_torchbind(self);
+          },
+          // __setstate__
+          [](GPT2BPEEncoderStatesTorchbind states)
+              -> c10::intrusive_ptr<GPT2BPEEncoder> {
+            return _deserialize_gpt2_bpe_encoder_torchbind(states);
+          });
+
   m.def("torchtext::generate_sp_model", &generate_sp_model);
   m.def("torchtext::load_sp_model", &load_sp_model);
   m.def("torchtext::load_sp_model_string", &load_sp_model_string);
   m.def("torchtext::gpt2_bpe_pre_tokenizer", &gpt2_bpe_pre_tokenizer);
```

Comment:
I guess we no longer need to bind this method right?

Reply:
We need to keep this method bound for internal GPT2 implementation that currently calls this operator. We can remove it later after internal dependency is removed and everything internally is migrated to the OSS tokenizer.